client/src/components/NewShowForm.js

```diff
@@ -1,12 +1,13 @@
 import React, { useState } from "react"
 import { Redirect } from "react-router-dom"
+
 import ErrorList from "./ErrorList"
 import translateServerErrors from "../services/translateServerErrors"
 
 const NewShowForm = (props) => {
   const [newShow, setNewShow] = useState({
     name: "",
-    description: ""
+    description: "",
```

Comment:
this addition with the extra comma seems...not good?

Reply:
^^ Nick's comment - trailing commas are sometimes used in formatting JS, but it is not consistent in this code base, I would remove